docker/devnet/boost/entrypoint.sh

```diff
@@ -45,6 +45,9 @@ if [ ! -f $BOOST_PATH/.init.boost ]; then
 	touch $BOOST_PATH/.init.boost
 fi
 
+# TODO(anteva): fixme: hack as boostd fails to start without this dir
+mkdir -p /var/lib/boost/deal-staging
```

Comment:
We need to fix this before we release LID right? Otherwise people will need to create this directory manually?

Reply:
Yeah, since we moved in some of the LID code to main we needed to add the line. Not sure what the root cause of it is.